src/PHPUnit/Constraint/Crawler.php

```diff
@@ -45,26 +56,30 @@ protected function fail($nodes, $selector, ComparisonFailure $comparisonFailure
         }
         $output .= "\ncontains text '{$this->string}'";
 
-        throw new \PHPUnit\Framework\ExpectationFailedException(
+        throw new ExpectationFailedException(
             $output,
             $comparisonFailure
         );
     }
 
-    protected function failureDescription($other) : string
+    /**
+     * @param mixed $other
+     * @return string
+     */
+    protected function failureDescription($other): string
```

Comment:
There is very specific `$o->textContent` in this method, so I guess that $other is an array of DOMElement